README.md

```diff
@@ -46,6 +50,29 @@ finding it at a path like `/usr/lib/x86_64-linux-gnu/qt5/bin/qmake`.
 Alternatively, you might want to load `goldendict.pro` file from within Qt Creator, especially on Windows.
 
 Note: To compile with `libhunspell` older than 1.5 pass `"CONFIG+=old_hunspell"` to `qmake`.
+## How to build on macOS Ventura 13.1 (succeeded on a Macbook Air M2)
+The old `*.dylib` files included in the `maclibs` directory are outdated (for x86), so they've been removed.
```

Comment:
Do these old files work with older architectures or Mac versions? If so, can the *maclibs/* directory stay? Depending on the detected Mac version or on a new `CONFIG` variable, *goldendict.pro* could either use *maclibs/* or rely on homebrew.